test/astable_flag.jl

```diff
@@ -120,5 +120,50 @@ end
     @test d == DataFrame(x = 1, z = 6)
 end
 
+@testset "grouping astable flag" begin
+    df = DataFrame(a = [1, 1, 2, 2], b = [5, 6, 7, 8])
+
+    gd = groupby(df, :a)
+
+    d = @combine gd @astable begin
+        ex = extrema(:b)
+        :b_min = ex[1]
+        :b_max = ex[2]
+    end
+
+    @test sort(d.b_min) == [5, 7]
```

Comment:
This kind of test is quite fragile. Better sort the whole data frame and compare it to the reference to make sure that groups and values match.